### PR TITLE
TSK-02 branch push

### PR DESCRIPTION
**Pull request by Szabo-Cosmin**
- created new branch

### CODE REVIEW

File: Basic FE project template/index.html [hunk 1]
```diff
@@ -10,9 +10,7 @@
   </head>
 
   <body>
-    <main>
-      <div>pateu gras</div>
-    </main>
+    <main></main>
```
Comment (torjescu) [inline Basic FE project template/index.html]: test comment on commit

Comment (Szabo-Cosmin) [reply Basic FE project template/index.html]: test replay

File: Basic FE project template/ui-src/styles/components/index.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+body {
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/components/index.scss]: why did you add code in an index.scss file? this should only contain file imports

Comment (torjescu) [reply Basic FE project template/ui-src/styles/components/index.scss]: also it is not correct to add it on the body

File: Basic FE project template/index.html [hunk 3]
```diff
@@ -11,10 +11,56 @@
...
+          <div class="why-beleasing__description">
+            <p>
+              Nullam suspendisse mauris blandit cum pharetra odio mauris lacus
+              risus. Velit at nam venenatis nunc lorem aliquam.
+            </p>
+          </div>
+          <ul class="advantages">
+            <li class="advantages__item">
+              <img
+                src="ui-src/assets/images/beleasing2-home-icon1.svg"
```
Comment (torjescu) [inline Basic FE project template/index.html]: change the src "ui-src/assets/images/beleasing2-home-icon1.svg" to "ui/assets/images/beleasing2-home-icon1.svg". The "ui' folder is the live folder, the "ui-src" folder is for development purposes.

File: Basic FE project template/index.html [hunk 4]
```diff
@@ -11,10 +11,56 @@
...
+                alt="icon"
+              />
+              <h2>Safety and security</h2>
+
+              <p>Egestas faucibus dui sit sagittis mi. Leo tempusinit</p>
+            </li>
+
+            <li class="advantages__item">
+              <img
+                src="ui-src/assets/images/beleasing2-home-icon2.svg"
```
Comment (torjescu) [inline Basic FE project template/index.html]: same here

File: Basic FE project template/index.html [hunk 5]
```diff
@@ -11,10 +11,56 @@
...
+                alt="icon"
+              />
+              <h2>Guarantee</h2>
+
+              <p>Mi ultrices lobortis nullanidun tincidunt non ante in.</p>
+            </li>
+
+            <li class="advantages__item">
+              <img
+                src="ui-src/assets/images/beleasing2-home-icon3.svg"
```
Comment (torjescu) [inline Basic FE project template/index.html]: same here

File: Basic FE project template/index.html [hunk 6]
```diff
@@ -11,10 +11,56 @@
...
+                alt="icon"
+              />
+              <h2>Possibility of return</h2>
+
+              <p>Dignissim et cras consequat nascetur bulla tinci</p>
+            </li>
+          </ul>
+          <div>
+            <img
+              src="ui-src/assets/images/beleasing2-home-pic4-500x435.webp"
```
Comment (torjescu) [inline Basic FE project template/index.html]: same here

File: Basic FE project template/index.html [hunk 7]
```diff
@@ -11,10 +11,56 @@
...
+            <p>
+              Nullam suspendisse mauris blandit cum pharetra odio mauris lacus
+              risus. Velit at nam venenatis nunc lorem aliquam.
+            </p>
+          </div>
+          <ul class="advantages">
+            <li class="advantages__item">
+              <img
+                src="ui-src/assets/images/beleasing2-home-icon1.svg"
+                alt="icon"
```
Comment (torjescu) [inline Basic FE project template/index.html]: alt="icon" is too generic, and you should not have alt's with the same values. you can change this to home icon

File: Basic FE project template/index.html [hunk 8]
```diff
@@ -11,10 +11,56 @@
...
+              />
+              <h2>Safety and security</h2>
+
+              <p>Egestas faucibus dui sit sagittis mi. Leo tempusinit</p>
+            </li>
+
+            <li class="advantages__item">
+              <img
+                src="ui-src/assets/images/beleasing2-home-icon2.svg"
+                alt="icon"
```
Comment (torjescu) [inline Basic FE project template/index.html]: change this alt as well

File: Basic FE project template/index.html [hunk 9]
```diff
@@ -11,10 +11,56 @@
...
+              />
+              <h2>Guarantee</h2>
+
+              <p>Mi ultrices lobortis nullanidun tincidunt non ante in.</p>
+            </li>
+
+            <li class="advantages__item">
+              <img
+                src="ui-src/assets/images/beleasing2-home-icon3.svg"
+                alt="icon"
```
Comment (torjescu) [inline Basic FE project template/index.html]: change this alt as well

File: Basic FE project template/index.html [hunk 10]
```diff
@@ -11,10 +11,56 @@
...
+              />
+              <h2>Possibility of return</h2>
+
+              <p>Dignissim et cras consequat nascetur bulla tinci</p>
+            </li>
+          </ul>
+          <div>
+            <img
+              src="ui-src/assets/images/beleasing2-home-pic4-500x435.webp"
+              alt="img of two people"
```
Comment (torjescu) [inline Basic FE project template/index.html]: change this alt as well

File: Basic FE project template/index.html [hunk 11]
```diff
@@ -5,16 +5,84 @@
...
 
     <link rel="stylesheet" type="text/css" href="ui/css/main.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Syne:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
 
     <title>Website title</title>
```
Comment (torjescu) [inline Basic FE project template/index.html]: give a title to the project

File: Basic FE project template/index.html [hunk 12]
```diff
@@ -5,16 +5,84 @@
...
 
   <body>
     <main>
-      <div>pateu gras</div>
-      <div>
-        <img src="ui-src/assets/images/audicar.webp" alt="car" />
-      </div>
+      <section class="why-beleasing">
+        <div class="container">
+          <div class="why-beleasing-flex">
```
Comment (torjescu) [inline Basic FE project template/index.html]: why-beleasing__container

File: Basic FE project template/ui-src/assets/images/beleasing2-home-icon1.svg
Comment (torjescu) [inline Basic FE project template/ui-src/assets/images/beleasing2-home-icon1.svg:1]: please create a folder in ui-src/assets/icons and place the icons there. Also create it in the ui/assets/icons path.

File: Basic FE project template/ui-src/styles/general/global/_global.scss [hunk 13]
```diff
@@ -1,3 +1,33 @@
-div {
-  color: red;
+* {
+  margin: 0;
+  top: 0;
+  box-sizing: border-box;
+}
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/general/global/_global.scss:5]: add empty line

File: Basic FE project template/ui-src/styles/general/global/_global.scss [hunk 14]
```diff
@@ -1,3 +1,33 @@
...
+.container {
+  width: 1380px;
+  margin: auto;
+  margin-top: 100px;
+  margin-bottom: 100px;
+  font-family: "Syne", sans-serif;
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/general/global/_global.scss]: add the font-family line of code to the body

File: Basic FE project template/ui-src/styles/general/global/_global.scss [hunk 15]
```diff
@@ -1,3 +1,33 @@
-div {
-  color: red;
+* {
+  margin: 0;
+  top: 0;
+  box-sizing: border-box;
+}
+.container {
+  width: 1380px;
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/general/global/_global.scss]: change witdth to 100%, add max-width: 1380px; and move the margins to the why-beleasing section

File: Basic FE project template/ui-src/styles/general/variables/_variables.scss [hunk 16]
```diff
@@ -1,3 +1,6 @@
 // Color variables
-
+:root {
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/general/variables/_variables.scss]: these are CSS variables, we need to use SCSS variables: $color--primary: #16182d; etc.

File: Basic FE project template/ui-src/styles/general/global/_global.scss [hunk 17]
```diff
@@ -1,3 +1,33 @@
...
+  color: var(--primary-color);
+  font-size: 48px;
+  line-height: 53px;
+  font-weight: 600;
+  letter-spacing: -2px;
+}
+
+h5 {
+  font-size: 19px;
+  color: var(--primary-color);
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/general/global/_global.scss]: sass variables will looks like this: color:  $color--primary;

File: Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss [hunk 18]
```diff
@@ -0,0 +1,65 @@
+.why-beleasing-flex {
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss]: rewrite scss: https://codepen.io/torjescu-sergiu/pen/JjeyrBN

File: Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss [hunk 19]
```diff
@@ -0,0 +1,65 @@
...
+
+.advantages {
+  display: flex;
+  list-style-type: none;
+  align-items: start;
+  padding: 0px;
+}
+
+.advantages__item {
+  margin-bottom: 40px;
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss]: write this in one line

File: Basic FE project template/index.html [hunk 20]
```diff
@@ -5,16 +5,84 @@
...
+              <img
+                src="ui/assets/images/beleasing2-home-pic4-500x435.webp"
+                alt="home-pic4"
+              />
+            </div>
+          </div>
+        </div>
+      </section>
     </main>
   </body>
```
Comment (torjescu) [inline Basic FE project template/index.html:88]: + responsive

File: Basic FE project template/index.html [hunk 21]
```diff
@@ -5,16 +5,85 @@
...
+    <link rel="stylesheet" href="ui-src/styles/queries.scss" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Syne:wght@400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
 
-    <title>Website title</title>
+    <title>Car Saleing</title>
```
Comment (torjescu) [inline Basic FE project template/index.html]: Saleing does not exist, Leasing

File: Basic FE project template/index.html [hunk 22]
```diff
@@ -5,16 +5,85 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
     <link rel="stylesheet" type="text/css" href="ui/css/main.css" />
+    <link rel="stylesheet" href="ui-src/styles/queries.scss" />
```
Comment (torjescu) [inline Basic FE project template/index.html]: remove the scss file from the html. why is it here?

File: Basic FE project template/ui-src/styles/queries.scss [hunk 23]
```diff
@@ -0,0 +1,107 @@
+@media screen and (max-width: 1238px) {
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/queries.scss]: this is not correct, we need to change everything. this is css

File: Basic FE project template/ui-src/styles/general/variables/_variables.scss [hunk 24]
```diff
@@ -1,3 +1,5 @@
 // Color variables
-
+$color--primary: #16182d;
+$-color--secondary: #676a70;
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/general/variables/_variables.scss]: change the names to $color--secondary and $color--border

File: Basic FE project template/ui-src/styles/style.scss [hunk 25]
```diff
@@ -1,2 +1,4 @@
-@import 'general/index';
+@import "general/index";
 @import "components/index";
+@import "components/why-beleasing/why-beleasing";
+@import "queries.scss";
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/style.scss]: this needs to be removed, we do not need a file named queries, each component will have it's own queries

File: Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss [hunk 26]
```diff
@@ -0,0 +1,70 @@
...
+
+  &__title {
+    max-width: 980px;
+    width: 100%;
+    margin-right: 10%;
+    margin-top: 30px;
+  }
+}
+
+.why-beleasing__description {
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss:63]: you can move this one after line 16

File: Basic FE project template/index.html [hunk 27]
```diff
@@ -5,16 +5,85 @@
...
+              </div>
+              <div class="why-beleasing__description">
+                <p class="description">
+                  Nullam suspendisse mauris blandit cum pharetra odio mauris
+                  lacus risus. Velit at nam venenatis nunc lorem aliquam.
+                </p>
+              </div>
+              <ul class="advantages">
+                <li class="advantages__item">
+                  <div class="home--icon1">
```
Comment (torjescu) [inline Basic FE project template/index.html]: change the class names to home-icon1, home-icon2 and home-icon3, and don't forget to change the names in the scss files as well if needed.

File: Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss [hunk 28]
```diff
@@ -0,0 +1,159 @@
...
+
+    @media screen and (max-width: 1238px) {
+      font-size: 16px;
+      line-height: 23px;
+      letter-spacing: 0px;
+    }
+  }
+}
+
+.advantages__item:hover,
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss:143]: you can move the :hover and :active after line 87 with &:active, &:hover

File: Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss [hunk 29]
```diff
@@ -0,0 +1,159 @@
+.why-beleasing {
+  margin: 100px 100px;
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss:2]: this can be written as -> margin: 100px;

File: Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss [hunk 30]
```diff
@@ -0,0 +1,159 @@
...
+    margin-right: 10%;
+    margin-top: 30px;
+
+    @media screen and (max-width: 1000px) {
+      max-width: 500px;
+      margin: 25px 0;
+      text-align: center;
+    }
+
+    @media screen and (max-width: 530px) {
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss:44]: as you have text-align: center; at line 41 in the media querie for 1000px, I think we do not need it here at 530px or at 375 px, so I think we can delete the 530px and 375px media queries below.

File: Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss [hunk 31]
```diff
@@ -0,0 +1,159 @@
...
+      text-align: center;
+    }
+
+    @media screen and (max-width: 375px) {
+      text-align: center;
+    }
+  }
+
+  p {
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss:53]: this looks like a global style because you targeted a "p" element. We have 2 options here: 1) we can move these styles int othe _globals.scss styles if you think this will be needed throughout the entire project, or 2) we can give this particular "p" a class in html and change this "p" here into that html class.

File: Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss [hunk 32]
```diff
@@ -0,0 +1,159 @@
...
+.advantages__item:hover,
+.advantages__item:active {
+  transition: all 0.3s;
+  animation-name: floating;
+  animation-duration: 1s;
+  animation-iteration-count: one;
+  animation-timing-function: ease-in;
+}
+
+@keyframes floating {
```
Comment (torjescu) [inline Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss:152]: we can do something cool here: We can create a folder called utils afer the general folder in styles. Inside the new util folder, we can create a _keyframes.scss file and copy the code there, we could reuse it in other components later if needed. Do not forget to import the newly created _keyframes.scss file inside the style.scss before the _why-beleaseing.scss file. (the order matters, because the keyframes are used inside the components, if we import them after the components, we will get errors).
